docs/installation/indexing-data/step-guide.rst

```diff
@@ -72,16 +72,9 @@ for searching, querying and accessing the data.
 The data from Geoscience Australia already comes with relevant files (named ``ga-metadata.yaml``), so
 no further steps are required for indexing them.
 
-For third party datasets, see :ref:`prepare-scripts`.
-
-
-.. admonition:: Note
-
-  :class: info
-
-    Some metadata requires cleanup before they are ready to be loaded.
-
-For more information see :ref:`dataset-metadata-doc`.
+For third party datasets, see the examples detailed `here <https://github.com/opendatacube/datacube-dataset-config#documented-examples>`__.
+For common distribution formations, data can be indexed using one of the tools from `odc-apps-dc-tools <https://github.com/opendatacube/odc-tools/tree/develop/apps/dc_tools>`__.
```

Comment:
"Distribution formations" sounds wrong - I think you mean "distribution formats"?